internal/charmstore/store.go

```diff
@@ -29,6 +29,11 @@ import (
 
 var logger = loggo.GetLogger("charmstore.internal.charmstore")
 
+const (
+	// StableChannel is the channel used internally for stable charms or bundles.
+	StableChannel charm.Channel = "stable"
```

Comment:
I've chosen to define them here for charmstore, that way we can remove it from charm later.